Data Cleaning/README.md

```diff
@@ -0,0 +1 @@
+The scripts in this folder read the raw version of the Health Survey of England data from `inputs\data` and save a cleaned version in `outputs\data`.
```

Comment:
It would be good to have a bit more info here on what each of the scripts do and a more general overview of the data cleaning tasks along with a title.

Reply:
Also make it clear if there is an order to running these scripts?

Reply:
Running the file `hse_1991.R` gives the error message:

```
Error in file(file, "rt") : cannot open the connection
In addition: Warning message:
In file(file, "rt") :
  cannot open file '/Users/isabel.stewart/AHL_r/ahl_weight_loss_modelling/inputs/data/1991_2009hse.tab': No such file or directory
```

Looking in inputs/data I can only see the file `hse_2019_eul_20211006.tab`